server/src/main/java/no/nav/foreldrepenger/vtp/server/api/pdl/PdlLeesahRestTjeneste.java

```diff
@@ -113,9 +147,36 @@ public Response leggTilHendelse(PersonhendelseDto personhendelseDto) {
             personhendelse.set("foedselsdato", fødselsdato.build());
         }
 
+        LOG.info("Publiserer FOEDSELSDATO_V1 på kafka for barn med ident {}, født: {}", barnIdent, fødselshendelseDto.fødselsdato());
         sendHendelsePåKafka(personhendelse.build());
+
+        produserForelderBarnRelasjon(fødselshendelseDto, barnIdent);
```

Comment:
API-et inn til denne klassen virker å være relativt teknisk orientert mot hendelser for PDL. Derfor er er kanskje litt overraskende at å bestille fødselshendelse også medfører forelder-barn-relasjonshendelser. 

Det er kanskje renere å håndtere trigging av forelder-barn-relasjonshendelsene fra utsiden, evt. ha en egen metode man kan bruke isdf leggTilHendelse som fyrer både fødselshendelse og relasjonshendelsene?

Reply:
evt styre det med et flagg.

Reply:
Ja, var litt usikker på om den burde også publisere flere hendelser her.
Fødselshendelser inneholder også fødselsdato, noe forelder-barn-relasjon ikke inneholder.
Det hjelper testing, og reduserer ekstra restkall.

Men kan toggle det med et flagg slik at man kan opte-in :)

Reply:
Hvis reell oppførsel er publisering umiddelbart etter Fødsel får dere vurdere om tester skal kalle 2 ganger eller at "fødeBarn" sender både Fødsel og ForelderBarn.
Reell oppførsel er også at ForelderBarn kan komme flere uker etter fødsel - først når relasjonen er registrert i FREG - etter at man har vært innom farskapsportalen og erklært farskap (dersom ikke gjøre før fødsel)

Reply:
Nei, mente ikke umiddelbart.
Det at fødselshendelse også publiserer forelder-barn-relasjon er nice-to-have feature som gjør testing noe lettere, og som er reelt mtp. rekkefølge av hendelser, ikke tidsrommet mellom dem. 

Det er også støtte for å publisere forelder-barn-relasjon hendelsene separat gjennom REST om man trenger å legge noe fiktive delays mellom dem.

Jeg kan legge på en queryParam for å opt-in på at de hendelsene samtidig publiserer forelder-barn-relasjon hendelser også, slik at ikke alle som bruker dette endepunktet får disse hendelsene servert om de ikke ønsker det.